cmake/thirdparty/SetupCUDA.cmake

```diff
@@ -158,14 +158,4 @@ blt_register_library(NAME          cuda
 # macros
 blt_register_library(NAME      cuda_runtime
                      INCLUDES  ${CUDA_INCLUDE_DIRS}
-                     LIBRARIES ${CUDA_LIBRARIES})
-
-# Allow the removal of implicitly found link directories
-# Note: This is a very specific fix for working around CMake adding implicit link
-# directories returned by the BlueOS compilers to link CUDA executables
-# They are added by logic found in the CMake source: Modules/CMakeTestCUDACompiler.cmake
-if(BLT_CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES_EXCLUDE)
```

Comment:
Folks may still be relying on the old version?

Reply:
I agree -- it is subtle.  Maybe add an error message and fail the config?

Reply:
I'm not sure anyone actually used the old version.. old being very relative.  I used it in serac very briefly but then we got a new set of compilers that didn't rely on gcc 4.9.3.

Reply:
I think there was another usage of this variable.  See #393.

Reply:
Nevermind, this is what I was thinking about: 
https://github.com/LLNL/blt/blob/ae1b0c281fbca0dc3f6afab8d7e5f078f754c162/cmake/thirdparty/SetupCUDA.cmake#L115-L116